### PR TITLE
Organization and additions

### PR DESCRIPTION
**Pull request by HTMLGhozt**
Organized JavaScript into subheadings.
- Books
- Documentation/Courses
- Libraries
- Resources

Added:
- Design > Assorted > [growth.design](https://growth.design/)
- JavaScript > Books > [Deep JavaScript](https://exploringjs.com/deep-js/toc.html)

Corrected Typo.

### CODE REVIEW

File: tips/technology-resources.md [hunk 1]
```diff
@@ -118,6 +118,7 @@
 #### Assorted
 
 - [freebiesbug.com](https://freebiesbug.com/) - A list of great, random free design resources
+- [growth.design](https://growth.design/) - UX case studies in a comic book format
```
Comment (nwthomas) [inline tips/technology-resources.md:121]: Nice addition!

File: tips/technology-resources.md [hunk 2]
```diff
@@ -222,28 +223,40 @@
 
 ## Javascript
 
-- [Arrayzing](https://gist.github.com/ourmaninamsterdam/1be9a5590c9cf4a0ab42#user-content-create-an-array) - Gives you answer on how to manage arrays by indexing topics from what you want to do, not the method names
+#### Books
```
Comment (nwthomas) [inline tips/technology-resources.md:226]: I think, in the context of an expanding section, sub-sections make sense here (especially for something as broad as JavaScript).

File: tips/technology-resources.md [hunk 3]
```diff
@@ -222,28 +223,40 @@
...
 - [JavaScript Spessore](https://github.com/raganwald/javascript-spessore) - A follow up for [JavaScript Allongé](https://leanpub.com/javascriptallongesix), it dives deeper into similar concepts, not strictly about functional programming
-- [JavaScript Visualizer](https://tylermcginnis.com/javascript-visualizer/) - A JavaScript visualizer that focuses on execution context
+- [Professor Frisby’s Mostly Adequate Guide to Functional Programming](https://github.com/MostlyAdequate/mostly-adequate-guide) - A deep dive into functional programming in JavaScript
+- [You Don't Know JS by Kyle Simpson](https://github.com/getify/You-Dont-Know-JS) - A series of JavaScript books that cover from 'Getting Started' to 'ES6 and Beyond'
+
+#### Documentation/Courses
+
+- [ECMA-262](https://tc39.github.io/ecma262/) - The defacto living standards guide for JavaScript
 - [The Modern JavaScript Tutorial](https://javascript.info/) - Free tutorials covering most relevant JavaScript topics
+- [Rithm School Intermediate JavaScript](https://www.rithmschool.com/courses/intermediate-javascript) - A course moving past the basics of JavaScript|
```
Comment (nwthomas) [inline tips/technology-resources.md:243]: Very cool - One of my coworkers went to Rithm.

File: tips/technology-resources.md [hunk 4]
```diff
@@ -361,4 +374,5 @@
 - [npkill](https://npkill.js.org/) - find and remove node_modules folders
 - [The Odin Project (Full Stack Bootcamp)](https://www.theodinproject.com/) - A full-fledged free coding bootcamp
 - [Postman Learning Resources](https://learning.getpostman.com/) - A great training center to understand how to use postman, essential for back-end work
+- [Startup School Library](https://www.startupschool.org/library) - Resources and previously taught lessions from Y Combinator's [Startup School](https://www.startupschool.org/)
```
Comment (nwthomas) [inline tips/technology-resources.md:377]: _Very_ nice addition.

File: tips/technology-resources.md (hunk 3 above)
Comment (HTMLGhozt) [reply tips/technology-resources.md:243]: I just found out Rithm supports Springboard Software Engineering. It's all connected man.

Comment (nwthomas) [reply tips/technology-resources.md:243]: :big-brain:

**Pull request merged**